_config.yml

```diff
@@ -98,7 +98,12 @@ defaults:
      authorimageurl: "/assets/images/bg.jpg"
      comments: true
 
-
+exclude: 
+  - README.md
+  - Gemfile
+  - Gemfile.lock
+  - CNAME
```

Comment:
Ignoring the CNAME might be risky, but if the site breaks after merge. I'll know why...